test/HtmlUtils-test.tsx

```diff
@@ -33,27 +33,33 @@ describe('HtmlUtils', () => {
     it('converts plain text topic to HTML', () => {
         const component = mount(<div>{ topicToHtml("pizza", null, null, false) }</div>);
         const wrapper = component.render();
-        expect(wrapper.text()).toEqual("pizza");
+        expect(wrapper.children().first().html()).toEqual("pizza");
     });
 
     it('converts plain text topic with emoji to HTML', () => {
```

Comment:
sorry, I meant a test where we send `"<b>pizza</b>"` through as plain text (and `null` html) to ensure it comes out as a literal.